### PR TITLE
Add Azure Pipelines build badge

### PR DESCRIPTION
**Pull request by kaylangan**
Adding a build badge for Azure Pipelines, scoped to `master`. Not sure if no branch scoping would be preferred so that this can be added to `master` in addition to `stable`?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,4 +13,4 @@ information.
 
 ### Subsystem maintainers
 
-* GHCJS - [Matchwood](https://github.com/matchwood)
\ No newline at end of file
+* GHCJS - [Matchwood](https://github.com/matchwood)
```
Comment (dbaynard) [inline README.md]: Think this may be a newline issue — can you fix, please?

Comment (kaylangan) [reply README.md]: Sure thing!

**Comment by mattaudesse** [comment]
Cool, thanks for this @kaylangan.

@dbaynard I know the PR's still technically waiting for your approval but since it's such a simple change I hope you won't mind me merging - sorry if I'm jumping the gun!

**Comment by dbaynard** [comment]
@mattaudesse thanks — perhaps we should propose a 'waiting for triager' label, for this kind of thing

**Pull request merged**